### PR TITLE
Milestone 2: Blog App | Add associations

### PR DESCRIPTION
**Pull request by kendoriddy**
**Blog App milestone 2: Processing Data in Models**

In this milestone I...

- Set up associations between models.
- Create model classes for all entities as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- Written custom methods for models.
- Written SQL queries with Active Record.
- Familiarized myself with [rails c command](https://guides.rubyonrails.org/command_line.html#bin-rails-console).
- Tested the models in a sandbox via rails c --sandbox

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
 end
```
Comment (muneebulrehman) [inline app/models/comment.rb]: Great job with the rest of the requirements. However,  the comment model should have a method that updates the comments counter for a post. Kindly add this method and submit it for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @kendoriddy , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good documentation.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/comment.rb (hunk 1 above)
Comment (kendoriddy) [reply app/models/comment.rb]: Changes requested have been made ✅ Thank you @muneebulrehman

**Review by danmainah — APPROVED** [review]
Hi @kendoriddy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kendoriddy** [comment]
Thank you @muneebulrehman and @danmainah for making my code even better, happy coding 😄 🚀

**Pull request merged**